lib/Runtime/HostManager/HostManager.cpp

```diff
@@ -113,6 +113,9 @@ llvm::Error HostManager::addNetwork(std::unique_ptr<Module> module,
   auto nodeList = std::move(partitioner.getPartitionResult());
 
   if (cctx.precisionConfig.quantMode == QuantizationMode::Profile) {
+    RETURN_ERR_IF_NOT(networks_.size() < 1,
```

Comment:
nit: I think RETURN_ERR_IF(networks_.size() != 0) might be a little easier to read.